xdsl/frontend/context.py

```diff
@@ -3,14 +3,15 @@
 from contextlib import AbstractContextManager
 from dataclasses import dataclass
 from inspect import getsource
-from sys import _getframe
+from sys import _getframe  #type: ignore
+from typing import Any  # type: ignore
 
 from xdsl.frontend.program import FrontendProgram
 from xdsl.frontend.python_code_check import PythonCodeCheck
 
 
 @dataclass
-class CodeContext(AbstractContextManager):
+class CodeContext(AbstractContextManager[Any]):
```

Comment:
No idea what the generic argument does, I did a quick search on GitHub and this was the first example of generics annotations on `AbstractContextManager` that I found...

Reply:
Yep, having a `Any` should always be fine!